### ISSUE TITLE
Call cloud function in cloud function, pass sessionToken not working 

### ISSUE THREAD

**Issue by flessard** [ParsePlatform/parse-server#363]
If you try to call an other cloud function inside a cloud function and you try to pass the sessionToken to get the user in the other cloud function. Not working. 

```
return Parse.Cloud.run('getUserRole', data, {
    sessionToken: request.user.get("sessionToken")
});
```

```
Parse.Cloud.define("getUserRole", function(request, response) {
    request.user // is undefined
})
```

**Comment by gfosco** [comment]
try `request.user.getSessionToken();` instead and let us know if that fixes it.

**Comment by flessard** [comment]
No, doesn't fix the problem.

**Comment by Tibabal** [comment]
@flessard Did you finally manage to get `request.user` working ( #269 ), or is it still only after fetching user data from server ?

**Comment by flessard** [comment]
@Tibabal i just make [pull request](https://github.com/ParsePlatform/parse-server/pull/373) to fix this [bug](https://github.com/ParsePlatform/parse-server/issues/269)

**Comment by gfosco** [comment]
Yep, this will be addressed in #373.
